store/tikv/tests/store_test.go

```diff
@@ -158,7 +158,11 @@ func (s *testStoreSuite) TestRequestPriority(c *C) {
 
 func (s *testStoreSerialSuite) TestOracleChangeByFailpoint(c *C) {
 	defer func() {
-		failpoint.Disable("github.com/pingcap/tidb/store/tikv/oracle/changeTSFromPD")
+		_, err := failpoint.Status("github.com/pingcap/tidb/store/tikv/oracle/changeTSFromPD")
```

Comment:
what if err happend?

Reply:
Let us remove this defer, or the `Disable` at L174. The `Disable` is called twice, so it reported error, right? @Tjianke